tests/snapshot_tests/snapshot_apps/scroll_page.py

```diff
@@ -0,0 +1,17 @@
+import os
+from textual.app import App
+from textual.widgets import RichLog
+
+
+class RichLogTest(App):
```

Comment:
Could you reduce the numbers here a bit? This test adds 2s to the test suite runtime on my machine.

Perhaps a small 5x10 terminal with smaller strings?

Reply:
Ok, reduced the snapshot size.